docker/candlepin-base/cp-test

```diff
@@ -174,7 +174,7 @@ if [ "$DEPLOY" == "1" ]; then
 
     DEPLOY_FLAGS="-g"
 
-    if [ "$RSPEC" == "1" ] && [ "$HOSTED" == "1" ]; then
+    if [ "$RSPEC" == "1" ] && [ "$HOSTED" == "1" ] && (bin/deploy -? | grep -q -- -h); then
```

Comment:
I'd probably quote the '-h' just to emphasis it's not an arg to grep. And maybe '-?' as well, just because unquoted punctuation in shell scripts looks suspicious (usage here isn't a problem though).